### ISSUE TITLE
On the last page of the demographics part of the test, the last question is inanswerable

### ISSUE THREAD

**Issue by worldpe** [SapplyValues/SapplyValues.github.io#9]
![2021-12-07T18:05:07,166852868-05:00](https://user-images.githubusercontent.com/82988612/145119752-03783f12-2aea-42de-855c-ee2fc84b3e39.png)

**Comment by worldpe** [comment]
Nothing occurs after pressing the buttons, and it makes the results of the test inaccessble.

**Comment by CIA-256** [comment]
Interesting. Anything under console/network tab when you answer the final question?

**Comment by egor83** [comment]
Getting this in the console:
https://i.imgur.com/kNxvBv5.png
then it takes me to the results page after a long pause.

**Comment by CIA-256** [comment]
Aha, CORS.
It shouldn't happen, but as we know, the world isn't ideal.
tl;dr it stalls your browser since it can't send the query to the datamining server because your browser doesn't like sending queries to different websites other than github.io, even though it theoretically should since I have set up the proper ways to allow it.

But, let's dive deeper.
What browser do you use? Any interesting add-ons installed?

**Comment by egor83** [comment]
Chrome on Linux (Ubuntu MATE).

Extensions - I don't think I have anything that interferes with CORS, but here's a full list:
ahfgeienlihckogmohjhadlkjgocpleb : Web Store : version 0_2
bkhaagjahfmjljalopjnoealnfndnagc : Octotree - GitHub code tree : version 7_4_3
chklaanhfefbnpoihckbnefhakgolnmc : JSONVue : version 0_1_17
cjpalhdlnbpafiamejdnhcphjbkeiagm : uBlock Origin : version 1_42_4
dpjamkmjmigaoobjbekmfgabipmfilij : Empty New Tab Page : version 1_2_0
fdpohaocaechififmbbbbbknoalclacl : GoFullPage - Full Page Screen Capture : version 7_6
fikfhablannhlgejpiklhbkonjkakodf : Search in background tab : version 0_0_6
fmkadmapgofadopljbjfkapdkoienihi : React Developer Tools : version 4_24_3
gfdkimpbcpahaombhbimeihdjnejgicl : Feedback : version 1_0
ghbmnnjooekpmoecnnnilnnbdlolhkhi : Google Docs Offline : version 1_42_0
hhinaapppaileiechjoiifaancjggfjm : Web Scrobbler : version 2_62_0
jcgpghgjhhahcefnfpbncdmhhddedhnk : Click to Remove Element : version 2_0_6
jlgkpaicikihijadgifklkbpdajbkhjo : CrxMouse Chrome™ Gestures : version 4_4_6
kbmfpngjjgdllneeigpgjifpgocmfgmb : Reddit Enhancement Suite : version 5_22_10
kmendfapggjehodndflmmgagdbamhnfd : CryptoTokenExtension : version 0_9_74
lcpkicdemehhmkjolekhlglljnkggfcf : imgur Uploader : version 2_4_0
mhjfbmdgcfjbbpaeojofohoefgiehjai : Chrome PDF Viewer : version 1
mhmphnocemakkjdampibehejoaleebpo : SuperNova SWF Enabler : version 0_88
mmeijimgabbpbgpdklnllpncmdofkcpn : Screencastify - Screen Video Recorder : version 2_69_0_4425
mnlaodleonmmfkdhfofamacceeikgecp : hckr news : version 2_0
neajdppkdcdipfabeoofebfddakdcjhd : Google Network Speech : version 1_0
nkeimhogjdpnpccoofpliimaahmaaome : Google Hangouts : version 1_3_18
nlbjncdgjeocebhnmkbbbdekmmmcbfjd : RSS Subscription Extension (by Google) : version 2_2_6
nmmhkkegccagdldgiimedpiccmgmieda : Chrome Web Store Payments : version 1_0_0_6
noaijdpnepcgjemiklgfkcfbkokogabh : ImTranslator: Translator, Dictionary, TTS : version 16_20
occjjkgifpmdgodlplnacmkejpdionan : AutoScroll : version 4_10
oeopbcgkkoapgobdbedcemjljbihmemj : Checker Plus for Gmail™ : version 23_0_5
omghfjlpggmjjaagoclmmobgdodcjboh : Browsec VPN - Free VPN for Chrome : version 3_50_0
pckkddchhdhkfapanbkcalloijbbjhbd : VK Blue - EQ & Scrobbler for VK.com : version 0_6_49

**Comment by CIA-256** [comment]
Alright, but does it change in incognito/private mode?

**Comment by egor83** [comment]
Nope, same delay, same error message, still shows results after a long pause.

**Comment by CIA-256** [comment]
I'll try to fiddle with the backend server later today, maybe I have something missing there.

**Comment by aleksejrs** [comment]
> The domain name censusbot.com is available for rent

**Comment by egor83** [comment]
The problem still exists.

**Comment by Jnorm911** [comment]
Is there a known working browser?

**Comment by MeowyCat1** [comment]
Can confirm, also occurs on Zen (Firefox based)

**Comment by MeowyCat1** [comment]
From looking, it appears that it is impossible to change CORS header on Github Pages, suggest either removing survey feature or setting up some kinda basic free proxy to inject headers
